requirements.txt

```diff
@@ -1,5 +1,5 @@
 alabaster==0.7.12
-aperturedb==0.1.1
+aperturedb==0.1.3
```

Comment:
this should always point to our latest version. can we just indicate that?

Reply:
Yea I think we don't care about versions and so many packages here too. I'll clean this up.
Good catch.